observability/logging/logging-6.1/log6x-release-notes-6.1.adoc

```diff
@@ -0,0 +1,81 @@
+:_mod-docs-content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
+[id="log6x-release-notes-6.1"]
+= Logging 6.0.0
+:context: logging-6x-6.1
+
+toc::[]
+
+This release includes link:https://access.redhat.com/errata/RHBA-2024:6693[{logging-uc} {for} Bug Fix Release 6.0.0]
```

Comment:
Wasn't sure, but do the components of this RN need to updated for 6.1? Maybe these are just placeholders and the updating hasn't happened yet, not sure.

Reply:
Please see my comment: https://github.com/openshift/openshift-docs/pull/84861#discussion_r1840475248